ui/main.qml

```diff
@@ -52,13 +47,17 @@ StatusWindow {
     title: {
         // Set application settings
         Qt.application.name = "Status Desktop"
-        Qt.application.displayName = qsTr("Status Desktop")
+        Qt.application.displayName = d.macOSWindowed ? "" : qsTr("Status Desktop")
```

Comment:
Keep it! Why this change? The display name should always stay the same

Reply:
this is required, else it looks like this 

<img width="1262" alt="image" src="https://github.com/user-attachments/assets/7c19cf24-7537-4aa7-b9d1-b091927d8923" />
 or maybe this is desired?